### PR TITLE
Submit and show November surveys

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

This PR adds the next survey modal, November. It submits or displays November (November) surveys. The last, major remaining bit in this feature set is #381 which is underway.

There's a lot of duplicate code between the surveys. The correct way would be that they'd inherit from a base survey component, but since we're nearly done with the survey feature and we wouldn't see the speed returns from doing that work, I've skipped it.

I did make some reusability and readability improvements, however, in creating multiple choice inputs.

Connects #380

### Demo
For good measure, here's a snippet of an existing November survey. 
screen shot 2019-01-09 at 1 07 15 pm

## Testing Instructions
Visit http://localhost:8000/survey/?beekeepers and log in, create 1+ apiaries.
Play with November surveys -- they should submit and show fine!

I made changes to April survey form too, to match November. To get an April survey, you'll have to update the `startYear` in utils from 2019 to say 2018 or 2017.

### CODE REVIEW

File: src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+            mite_management_CHEMICAL_THYMOL_MENTHOL_APIGUARD: false,
+            mite_management_CHEMICAL_SYNTHETIC_APIVAR: false,
+            mite_management_CHEMICAL_SYNTHETIC_APISTAN: false,
+            mite_management_CHEMICAL_SYNTHETIC_CHECKMITE_PLUS: false,
+            mite_management_MECHANICAL_DRONE_BROOD_REMOVAL: false,
+            mite_management_MECHANICAL_QUEEN_MANIPULATION: false,
+            completedSurvey: '',
+            error: '',
+        };
+        this.multipleChoiceKeys = [
```
Comment (rajadain) [inline src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx]: `varroa_check_method` is missing from `multipleChoiceKeys`

File: src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+                            value={num_colonies}
+                            disabled={!!completedSurvey}
+                            required
+                        />
+                        <div className="form__group">
+                            <label htmlFor="harvested_honey">
+                                How much honey did you collect on average for each colony?
+                            </label>
+                            <select
+                                type="text"
```
Comment (rajadain) [inline src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx]: `select`s don't have `type`

File: src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+                            checked={small_hive_beetles}
+                            onChange={this.handleChange}
+                            disabled={!!completedSurvey}
+                        />
+                        <div className="form__group">
+                            <label htmlFor="varroa_check_frequency">
+                                How often do you check for Varroa?
+                            </label>
+                            <select
+                                type="text"
```
Comment (rajadain) [inline src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx]: Here too

Comment (fungjj92) [reply src/icp/apps/beekeepers/js/src/components/NovemberSurveyForm.jsx]: oh cool. ill be sure to update in `UserSurveyModal` too

**Comment by rajadain** [comment]
Taking another look

**Comment by fungjj92** [comment]
Awesome, thanks for review. Onward to the last survey! Merging.

**Pull request merged**